src/samples.jl

```diff
@@ -40,6 +40,8 @@ struct Samples{D<:AbstractMatrix}
     end
 end
 
+Base.:(==)(a::Samples, b::Samples) = a.encoded == b.encoded && a.signal == b.signal && a.data == b.data
```

Comment:
I think this definition makes sense, but it should probably be documented somewhere. 

(Because this definition is whether two `Samples` are equivalent in their representation, but equivalence of  the two samples represented by two `Samples` would also be a potentially valid definition of `==`.)

Reply:
Done